packages/fulfillment/src/models/fulfillment.ts

```diff
@@ -88,28 +89,37 @@ export default class Fulfillment {
   @Property({ columnType: "jsonb", nullable: true })
   data: Record<string, unknown> | null = null
 
-  @Property({ columnType: "text" })
+  @ManyToOne(() => ServiceProvider, {
```

Comment:
Don't forget this one if we go with FulfillmentProvider 😄

Reply:
yes ofcourse thanks :) but since I test before pushing I can't miss it ahah